platform/ui-next/src/components/DataRow/DataRow.tsx

```diff
@@ -261,25 +261,28 @@ const DataRow: React.FC<DataRowProps> = ({
           </Button>
 
           {/* Lock Icon (if needed) */}
```

Comment:
I'm working on replacing the entire DataRow with customizeable pieces to allow customizing the menu, appearance etc.